.azure-pipelines.yml

```
@@ -129,7 +128,7 @@ jobs:
   - bash: |
       set -x -e
       source activate testing
-      gmt which -Gu @earth_relief_10m @earth_relief_01d @ridge.txt @Table_5_11.txt @tut_bathy.nc @tut_quakes.ngdc @tut_ship.xyz
+      gmt which -Ga @earth_relief_10m @earth_relief_01d @ridge.txt @Table_5_11.txt @tut_bathy.nc @tut_quakes.ngdc @tut_ship.xyz
```

```suggestion
      gmt which -Ga @earth_relief_10m_p @earth_relief_01d_p @earth_relief_10m_g @earth_relief_01d_g @ridge.txt @Table_5_11.txt @tut_bathy.nc @tut_quakes.ngdc @tut_ship.xyz
```

Will need to cache both the pixel and gridline grids now